lib/plaidRequest.js

```diff
@@ -28,6 +28,13 @@ var handleApiResponse = function(resolve, reject, err, res, $body, isMfa) {
 
   // success response (non mfa)
   } else if (res.statusCode === 200) {
+    // when Plaid-Request-Id exists we assume binary data are returned
+    if (res.headers['plaid-request-id']) {
```

Comment:
@dsfish @r-ohan is this a safe assumption to make? I feel like this may be the case only by coincidence.

A possible alternative might be to implement some sort of `PlaidBuffer` class that extends Buffer and has some method like `.RequestId()` available?

Reply:
@dsfish do we return a `Content-Type` header? That seems to be the ideal way of handling this. It doesn't look like we do from what I can tell. It would be a pretty easy fix to implement.

Reply:
@r-ohan you do return it :)

Reply:
Yeah eventually moving request_id's into the headers for all responses seems like something we'd want to to do, but I doubt that it's something we can tackle anytime soon. (cc @nathankot )

We shouldn't assume that if a request_id is sent in the header that the response type is binary, this doesn't seem like a future-proof solution. Can you add a check for `Content-Type` to line 32 as well? If we do decide to move request_id to headers it'll most likely come with a change to the client libs to handle the change.

Reply:
@r-ohan Sure, just to make sure we understand each other:
```javascript
if (res.headers['plaid-request-id'] && res.headers['content-type']==='application/pdf') {
```
correct?

Reply:
That looks good - I think we can go a step further and say that any content type starting with `application` will be a binary data type. [Explanation of the application mime type for reference.](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types#Discrete_types)

So it should look like this:

```node
if (res.headers['plaid-request-id'] && res.headers['content-type'] && res.headers['content-type'].indexOf('application') === 0) {
```

Thanks for your help with this!!

Reply:
Cool, good catch with the `application` mime-type 👍